language/az.php

```diff
@@ -584,7 +584,8 @@
     'Search Results' => false,
     'feed item' => false,
     'Allow handling of mailto links' => false,
-    'Show folders' => false
+    'Show folders' => false,
```

Comment:
@jasonmunro Do committers need to do this? Seems we shouldn't worry about this until a translator wants to work on a specific language.

In Tiki, we do like this: https://gitlab.com/tikiwiki/tiki/-/blob/master/lib/core/Tiki/Command/GetStringsCommand.php

Reply:
Actually I do like when this done, it makes it easier for a translator to be able to convert all the strings from a single source.